src/tensors/factorizations/utility.jl

```diff
@@ -0,0 +1,29 @@
+# convenience to set default
+macro check_space(x, V)
+    return esc(:($MatrixAlgebraKit.@check_size($x, $V, $space)))
+end
+macro check_scalar(x, y, op=:identity, eltype=:scalartype)
+    return esc(:($MatrixAlgebraKit.@check_scalar($x, $y, $op, $eltype)))
+end
+
+function factorisation_scalartype(t::AbstractTensorMap)
+    T = scalartype(t)
+    return promote_type(Float32, typeof(zero(T) / sqrt(abs2(one(T)))))
+end
+factorisation_scalartype(f, t) = factorisation_scalartype(t)
+
+function permutedcopy_oftype(t::AbstractTensorMap, T::Type{<:Number}, p::Index2Tuple)
```

Comment:
Is `permutedcopy_oftype` still used anywhere other than in the deprecations? If not, it could potentially also be moved to "src/auxiliary/deprecate.jl".